starboard/shared/starboard/application.cc

```diff
@@ -410,6 +410,10 @@ bool Application::DispatchAndDelete(Application::Event* event) {
 }
 
 bool Application::HandleEventAndUpdateState(Application::Event* event) {
+  // Don't accept any events in kStateStopped state
+  if (state_ == kStateStopped)
```

Comment:
@kaidokert is this ok to reject all events if the state is stopped?

Reply:
> @kaidokert is this ok to reject all events if the state is stopped?

When application goes to  state_ == kStateStopped it means that the button "Close YT" was pressed. It is the last event. Almost last, because the next, really last event is "Button Up" event. It does nothing, because system reacts on "Button Down" event. But application can be already destroyed when  last "Button Up" goes. It is the reason I ignore all events in kStateStopped state

Reply:
No, we have never seen this happen. We had an increased crash rate on the Xbox One platform that we were unable to reproduce. That’s why we reviewed all the code for the potential root of that problem. In this case we suspected that in some cases during the app exit a crash report may be generated. As @victorpasoshnikov [wrote](https://github.com/youtube/cobalt/pull/1524#discussion_r1417911462), now we know that this never happens, so this fix is not required.